friends/src/Components/friendsList.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class FriendsList extends Component {
+    constructor(){
+        super ();
+    this.state = {
+        friends:[],
+        friend:[]
+    }
+}
+
+        componentDidMount(){
+            axios.get('http://localhost:5000/friends')
+              .then(response => {
+                console.log(response);
+                this.setState({friends: response.data});
+              })
+              .catch(err => {
+                console.log(err);
+              });
+
+        this.onFormSubmit = e => {
+            e.preventDefault();
+            this.setState ({
```

Comment:
Make sure you're using your CRUD operations for this assignment! You'll need a `POST` request to the server here.